titlecheck.lic

```diff
@@ -0,0 +1,66 @@
+=begin
+This script requires that you have ;dependency installed... it will not work otherwise
+Checks for new titles since the last time you ran ;titlecheck.
+Usage: ;titlecheck <title set>  (e.g. ;titlecheck moonmage)
+
+author: Crannach
+game: dr
+=end
+
+class TitleCheck
+  include DRC
+  def initialize
+    arg_definitions = [
+        [
+        {name: 'title_set', options: %w(blunt ranged brawling genedged specedged thrown pole shield slings weapons weaponmaster performer primarymagic magic money ownership survival1 survival2 survival3 lore criminal generic racial premium order religion novice practitioner dilettante aficionado adept expert professional authority genius savant master grandmaster guru legend cleric trader warmage necromancer empath ranger paladin bard barbarian thief moonmage), description: 'What set of titles are you checking?'}
+      ] 
+    ]
+    args = parse_args(arg_definitions)
+    UserVars.titles = Hash.new if UserVars.titles.nil?
```

Comment:
💡 I _think_ this could also be written as `UserVars.titles |= {}` if you find that clearer

Reply:
`UserVars.titles ||= {}` rather?

Reply:
Yep, good catch